zebra/zebra_cli.h

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#ifndef _ZEBRA_CLI_H
+#define _ZEBRA_CLI_H 1
+
+extern const struct frr_yang_module_info frr_zebra_cli_info;
+
+void zebra_cli_init(void);
```

Comment:
I've been putting these 2 in `foobar_nb.h`. Having this separate file is certainly more correct, but ... it's an extra file for 2 symbols :)

Reply:
Yes, but I want to fully separate CLI from the other code. By including headers like `zebra_nb.h` we include a lot of function declarations that don't make sense in mgmtd. I understand that it doesn't affect the actual build in any way, but still seems messy.